syntax = "proto3";

package api;

import "api/api.proto";

service StockWaveService {
  rpc SortNumbers (NumberArray) returns (NumberArray) {}
}